README.md

```diff
@@ -11,30 +11,31 @@ The Diamond Light Source developer portal, built with backstage for deployment o
 
 ## Developing Locally (VSCode)
 
-- [Install docker-compose](https://docs.docker.com/compose/install/other/)
+- [Install docker-compose](https://docs.docker.com/compose/install/other/) or load from the module system using `module load docker-compose`
 - Start the podman socket with `systemctl --user start podman.socket`
 - Use the podman socket as the compose host by adding `export DOCKER_HOST=unix:///run/user/$UID/podman/podman.sock` to your `~/.bashrc`
```

Comment:
The module should cover all three of these, we should re-structure this to reflect that